python/lsst/cp/pipe/cpDark.py

```diff
@@ -73,6 +73,16 @@ class CpDarkTaskConfig(pipeBase.PipelineTaskConfig,
         target=RepairTask,
         doc="Repair task to use.",
     )
+    maskListToInterpolate = pexConfig.ListField(
+        dtype=str,
+        doc="List of mask planes that should be interpolated.",
+        default=['SAT', 'BAD'],
```

Comment:
I think the SAT is fine, but in a dark anything that is SAT better be marked BAD!